src/main/java/com/FlightPub/Controllers/IndexController.java

```diff
@@ -193,12 +193,13 @@ public String bookFlight(@RequestParam String id, @RequestParam Integer seats ,M
     @PostMapping("/search")
     public String runSearch(@ModelAttribute BasicSearch search, Model model, HttpSession session){
         model = addDateAndTimeToModel(model);
-        List<Flight> flights;
+        List<Flight>[] flights = new ArrayList[2];
         List<StopOver>[] stopOver = new ArrayList[3];
```

Comment:
Would making this an array list & not assuming the amount of stopovers provide any notable difference for any potential future changes, and also remove they try catch used later?